### ISSUE TITLE
add support for multiple clicks of the earphones button

### ISSUE THREAD

**Issue by TruncatedDinoSour** [FossifyOrg/Music-Player#44]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Music-Player/issues) or [closed](https://github.com/FossifyOrg/Music-Player/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Music-Player/discussions) or [closed](https://github.com/FossifyOrg/Music-Player/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Music-Player/blob/master/CONTRIBUTING.md).
- [ ] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

it's very annoying on wired earphones which don't have different buttons to be forced to use multiple buttons, simplemobiletools music player has support for this, so i assumed this has it too - i was wrong

### Why do you want this feature?

explained above

### Additional information

pressing it multiple times just stops the current track

**Comment by naveensingh** [comment]
> simplemobiletools music player has support for this

There is no such difference between the last open-source version of the SMP and the initial version of the FMP apart from the name.

**Comment by TruncatedDinoSour** [comment]
> > simplemobiletools music player has support for this
> 
> There is no such difference between the last open-source version of the SMP and the initial version of the FMP apart from the name.
> 
> Please describe the behavior you want to be implemented like what should happen when the button is clicked once, twice, or thrice.

fr ?

i installed SMP from fdroid and pressing the button multiple works fine
can't say the same about FMP, though

once : pause current track
twice : skip track
thrice : guess skip and pause ? i just tested it it just does that

**Comment by naveensingh** [comment]
> fr ?

If you don't count the name, icon, translations, and some library updates then yeah, there is no functionality change between the last SMP (5.18.3) and the current FMP version (1.0.0). 

I can't test at the moment but if this really is broken, it should be broken on SMP as well because the whole playback service was rewritten a while ago.

**Comment by TruncatedDinoSour** [comment]
> > fr ?
> 
> If you don't count the name, icon, translations, and some library updates then yeah, there is no functionality change between the last SMP (5.18.3) and the current FMP version (1.0.0).
> 
> I can't test at the moment but if this really is broken, it should be broken on SMP as well because the whole playback service was rewritten a while ago.

possibly ? as it works fine with SMP, but not FMP, best of luck <3

**Comment by poetter-sebastian** [comment]
Heya, it was the same for me when I changed the version from SMP to FMP.

**Comment by TruncatedDinoSour** [comment]
> Heya, I have the same behavior after I changed from SMP to FMP.

exactly, its weird, i installed SMP ( from fdroid ) and its fine now lmao, FMP wont work

**Comment by TruncatedDinoSour** [comment]
still not fixed, tested with the latest :)

it just pauses with double click, ill continue with SMP ig
